src/CodeExplorerBundle/Twig/SourceCodeExtension.php

```diff
@@ -99,15 +99,23 @@ private function getCallableReflector($callable)
 
     private function getTemplateSource(\Twig_Template $template)
     {
+        // Twig templates are not always stored in files, and so there is no
+        // API to get the filename from a template name in a generic way.
+        // The logic used here works only for templates stored in app/Resources/views
+        // and referenced via the "filename.html.twig" notation, not via the "::filename.html.twig"
+        // one or stored in bundles. This is enough for the needs of the demo app.
+        $filePath   = $this->kernelRootDir.'/Resources/views/'.$template->getTemplateName();
+        $sourceCode = $template->getSource();
+
+        // Temporary workaround for https://github.com/symfony/symfony-demo/issues/224
```

Comment:
Maybe we can change this URL and link to the Twig issue report: `https://github.com/twigphp/Twig/issues/2011`